src/components/Global/Filter/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import RadioButton from '../RadioButton';
+import styles from './Filter.module.scss';
+
+const Filter = () => {
+  const [easyChecked, setEasyChecked] = useState(false);
+  const [mediumChecked, setMediumChecked] = useState(false);
+  const [hardChecked, setHardChecked] = useState(false);
+
+  return (
+    <div>
+      <form className={styles.filter}>
```

Comment:
The `Filter` component should also contain an `<Input>` and the state of that input method should also be managed here. You'll probably need to merge the updated version (after resolving reviews) of the Input component into this branch again